regression/__snapshots__/validateExamplesRegression_latestOnly.test.ts.snap

```diff
@@ -1107,6 +1107,158 @@ Array [
     "severity": 0,
     "source": "response",
   },
+  Object {
```

Comment:
This file doesn't need to update and can you remove it from this PR?

Reply:
sorry, my fault, now this PR jsut modify the validateExamplesRegression_nonLatest.test.ts.snap